### ISSUE TITLE
Can't shim '\resources\cli\atom.cmd': File doesn't exist.

### ISSUE THREAD

**Issue by talha131** [lukesampson/scoop-extras#414]
I just updated atom using

```
scoop update atom
Updating 'atom' (1.16.0 -> 1.17.0)
Uninstalling 'atom' (1.16.0)
Removing shim for 'atom'.
Removing shim for 'apm'.
Unlinking ~\scoop\apps\atom\current
Installing 'atom' (1.17.0).
atom-windows.zip (115.5 MB) [=====] 100%
Checking hash of atom-windows.zip... ok.
Extracting... done.
Linking ~\scoop\apps\atom\current => ~\scoop\apps\atom\1.17.0
Creating shim for 'atom'.
Can't shim '\resources\cli\atom.cmd': File doesn't exist.
```

I think it is related #413 issue. Because `extract_dir` is not set. Atom package unarchives into a `Atom` folder inside `1.17.0` folder. 

1.16.0 folder has the correct contents. But `1.17.0` folder only has a single folder `Atom` which in turn has the actual content.

**Comment by r15ch13** [comment]
Hm, it looks like a bigger problem. If extract_dir is not set in the autoupdate then it will be empty on the next update. Normally there is one with a directory with a version, but not in these cases. Will look into it tomorrow.

**Comment by r15ch13** [comment]
Fixed with 6a79de68261064529a5a2e2a7da05f1ea9ea21d7
